src/pages/PodcastListPage/PodcastListPage.js

```diff
@@ -9,8 +9,15 @@ PodcastListPage.propTypes = {
 export default function PodcastListPage({ podcasts }) {
   return (
     <PageContainer>
-      {podcasts.map(({ image, title, id }) => (
-        <PodcastCard key={id} image={image} title={title} />
+      {podcasts.map(({ image, title, id, author, description, categories }) => (
+        <PodcastCard
+          key={id}
+          image={image}
+          title={title}
+          author={author}
+          description={description}
+          categories={categories ? categories : {}}
+        />
```

Comment:
You can reduce it here by using the spread operator.
This works, if categories is always set (maybe add it for every podcast?)

```suggestion
      {podcasts.map((podcast) => (
        <PodcastCard
          key={podcast.id}
          {...podcast}
        />
```

Reply:
Thank you for your great suggestion, I will try that. The Problem is that i have to catch an null object. The data im using has sometimes nul as value in the categories key.